client/src/components/layout/BuildsList.js

```diff
@@ -15,7 +17,7 @@ const BuildsList = (props) => {
         }
     }
 
-    const buildTiles = currentBuilds.map((build) => {
+    let buildTiles = currentBuilds.map((build) => {
```

Comment:
always prefer `const` over `let` unless reassigning, which doesn't seem to be the case here